spec/extensions/changes_spec.rb

```diff
@@ -273,7 +273,7 @@
         end
 
         form = form_class.new(:foo => "12345")
-        assert_equal(Fixnum, new_value_class)
+        assert_equal(Integer, new_value_class)
```

Comment:
https://weblog.rubyonrails.org/2016/10/8/this-week-in-rails-getting-ready-for-ruby-2-4/#:~:text=Fixnum%20and%20Bignum%20are%20deprecated,legacy%20classes%20were%20deprecated%20recently.